renderdoc/driver/vulkan/vk_serialise.cpp

```diff
@@ -1118,15 +1132,35 @@ SERIALISE_VK_HANDLES();
   PNEXT_UNSUPPORTED(VK_STRUCTURE_TYPE_PERFORMANCE_OVERRIDE_INFO_INTEL)                                 \
   PNEXT_UNSUPPORTED(VK_STRUCTURE_TYPE_PERFORMANCE_CONFIGURATION_ACQUIRE_INFO_INTEL)                    \
                                                                                                        \
+  /* VK_EXT_shader_image_atomic_int64 */                                                               \
+  PNEXT_UNSUPPORTED(VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_IMAGE_ATOMIC_INT64_FEATURES_EXT)          \
```

Comment:
This is nit-picky I know but I try to keep this list in alphabetical order since this file is a little unwieldy.  So this and `VK_EXT_shader_atomic_float` should go above `VK_EXT_texture_compression_astc_hdr` next to `VK_EXT_image_robustness`.